### ISSUE TITLE
Integration testing after applying catbuffer

### ISSUE THREAD

**Issue by fullcircle23** [nemtech/nem2-sdk-java#79]
Revisit E2E testing after #73 Apply Catbuffer.
- Fix Standalone and Aggregate Transfer Transaction

**Comment by fullcircle23** [comment]
E2ETest
Test standaloneTransferTransaction
Error:
java.lang.ClassCastException: java.lang.Integer cannot be cast to java.lang.Long
	at io.nem.sdk.infrastructure.Listener.extractBigInteger(Listener.java:313)

**Comment by fullcircle23** [comment]
The following E2E tests passed:
1. standaloneTransferTransaction 
2. aggregateTransferTransaction
3. standaloneRootRegisterNamespaceTransaction
4. aggregateRootRegisterNamespaceTransaction
5. standaloneSubNamespaceRegisterNamespaceTransaction
6. aggregateSubNamespaceRegisterNamespaceTransaction
7. standaloneMosaicDefinitionTransaction 
8. aggregateMosaicDefinitionTransaction
9. standaloneMosaicSupplyChangeTransaction 
10. aggregateMosaicSupplyChangeTransaction
11. standaloneLockFundsTransaction
12. aggregateLockFundsTransaction
13. standaloneSecretLockTransaction
14. aggregateSecretLockTransaction
15. standaloneSecretProofTransaction
16. aggregateSecretProofTransaction

Todo the following E2E tests:
1. aggregateBondedTransaction
2. aggregateBondedCosignatureTransaction

**Comment by fullcircle23** [comment]
E2E Transactions tests passed for the 16 tests listed above. Will do a PR and close this ticket first and open new ticket to add a few more tests for aggregateBonded and alias transactions.
